src/index.html

```diff
@@ -7,8 +7,198 @@
+            <div id="skilname_3" class="skilname">Microsoft</div>
+            <div id="skilprof_3" class="skilprof"></div>
+          </div>
+          <div id="skls_4" class="skls_total">
+            <div id="skilname_4" class="skilname">Microsoft</div>
+            <div id="skilprof_4" class="skilprof"></div>
+          </div>
+          <div id="skls_5" class="skls_total">
+            <div id="skilname_5" class="skilname">Microsoft</div>
+            <div id="skilprof_5" class="skilprof"></div>
+          </div>
+          <div id="skls_6" class="skls_total">
+            <div id="skilname_6" class="skilname">Microsoft</div>
+            <div id="skilprof_6" class="skilprof"></div>
+          </div>
+        </div>
+      </div>
+      <div id="conta">
+        <div id="quicid">
+          <div id="quicid_t">QUICK ID</div>
+          <hr />
+          <div id="quicid_n1" class="quicid_name"></div>
+          <div id="quicid_d1" class="quicid_detial"></div>
+          <div id="quicid_n2" class="quicid_name"></div>
+          <div id="quicid_d2" class="quicid_detial"></div>
+          <div id="quicid_n3" class="quicid_name"></div>
+          <div id="quicid_d3" class="quicid_detial"></div>
+          <div id="quicid_n4" class="quicid_name"></div>
+          <div id="quicid_d4" class="quicid_detial"></div>
+        </div>
+        <div id="contame">
+          <div id="quicid_t">QUICK ID</div>
+          <hr />
+          <form>
+            <label
+              >Name<input
+                type="text"
+                name="name"
+                placeholder="Name"
+                value=""
+                id="quicied_name"
+            /></label>
+            <label
+              >Email<input
+                type="text"
+                name="email"
+                placeholder="Email"
+                value=""
+                id="quicied_email"
+            /></label>
+            <label
+              >Subject<input
+                type="text"
+                name="subject"
+                placeholder="Subject"
+                value=""
+                id="quicied_subject"
+            /></label>
+            <label
+              >Message<input
+                type="text"
+                name="message"
+                placeholder="Message"
+                value=""
+                id="quicied_message"
+            /></label>
+            <input type="submit" value="Submit" />
+          </form>
+        </div>
+      </div>
+      <footer>
+        <button id="download">DOWNLOAD MY FULL RESUME</button>
+        <div id="copr">copy right</div>
+        <div id="fot_social">
+          <div id="linki_foot" class="socialM">
+            <a href="https://www.linkedin.com/"><img src="" alt="LinkIn"/></a>
+          </div>
+          <div id="twitt_foot" class="socialM">
+            <a href="https://www.twitter.com/"><img src="" alt="Twitter"/></a>
+          </div>
+          <div id="googl_foot" class="socialM">
```

Comment:
Please apply the same name convention for tag attributes. You are following snake_case for IDs but camelCase for class names, which obeys the consistency.